packages/core/lib/commands/install.js

```diff
@@ -3,33 +3,43 @@ const command = {
   description: "Install a package from the Ethereum Package Registry",
   builder: {},
   help: {
-    usage: "truffle install <package_name>[@<version>]",
+    usage: "truffle install <packageId> [--alias]",
     options: [
       {
-        option: "package_name",
-        description:
-          "Name of the package as listed in the Ethereum Package Registry. (required)"
+        option: "packageId",
+        description: `(required) Name of the package as listed in the Ethereum Package Registry. 
+          Accepted formats: packageName, packageName@version, ethpm URI, ipfs URI.`
       },
       {
-        option: "<@version>",
+        option: "--alias",
         description:
-          "When specified, will install a specific version of the package, otherwise " +
-          "will install\n                    the latest version."
+          "A different name under which this package will be installed."
       }
     ]
   },
-  run: function(options, done) {
+  run: async function (options, done) {
     const Config = require("@truffle/config");
-    const Package = require("../package");
+    const PackageV1 = require("@truffle/ethpm-v1");
+    const PackageV3 = require("@truffle/ethpm-v3");
 
-    if (options._ && options._.length > 0) options.packages = options._;
+    if (options._ && options._.length == 0) {
+      done(new Error(`Please provide a packageId.`));
```

Comment:
```suggestion
      done(new Error(`Please provide a package identifier.`));
```

Should also probably expand on this error with additional info, like, "Please see `truffle help install` for more information on valid package identifiers."